Make the details and creator/author fields optional when using the DAM license feature (#2376)
vivid-planet/comet: .changeset/pink-panther-purr.md

---
"@comet/cms-admin": minor
---

> Set license details and creator/author to be optional fields when using the dam file license feature

Reviewer: ```suggestion
Make the details and creator/author fields optional when using the DAM license feature
```
